docs/source/06_nodes_and_pipelines/03_modular_pipelines.md

```diff
@@ -415,54 +276,70 @@ final_pipeline = (
+ Mapping parameter values is very similar to the way we map inputs and outputs.
+
+<details>
+<summary>Click here to see a worked example</summary>
+
+* We instantiate the `template_pipeline` twice, but pass in different parameters.
+* `input1` and `input2` are 'frozen' and thus shared in both instances.
+* `params:override_me` does not actually exist and is designed to be overridden in both cases.
+* Providing a namespace isolates the intermediate operation and visualises nicely.
 
 ```python
-alpha_pipeline = Pipeline(
+template_pipeline = Pipeline(
     [
-        node(node_func1, ["input1", "input2", "params:alpha"], "intermediary_output"),
-        node(node_func2, "intermediary_output", "output"),
+        node(
+            func=node_func1, 
+            inputs=["input1", "input2", "params:override_me"], 
+            outputs="intermediary_output"
+        ),
+        node(
+            func=node_func2, 
+            inputs="intermediary_output", 
+            outputs="output"
+        ),
     ]
 )
+
+alpha_pipeline = pipeline(
+    pipe=template_pipeline,
+    inputs={"input1", "input2"},
+    parameters={"params:override_me": "params:alpha"}
+    namespace="alpha"
+)
+
 beta_pipeline = pipeline(
-    alpha_pipeline,
+    pipe=template_pipeline,
     inputs={"input1", "input2"},
-    parameters={"params:alpha": "params:beta"},
+    parameters={"params:override_me": "params:beta"},
     namespace="beta",
 )
 
 final_pipeline = alpha_pipeline + beta_pipeline
 ```
 
-The value of parameter `alpha` is replaced with the value of parameter `beta`, assuming they both live in your parameters configuration (`parameters.yml`). The namespace ensures that outputs are not overwritten, so intermediate and final outputs are prefixed, i.e. `beta.intermediary_output`, `beta.output`.
-
-## How to clean up a modular pipeline
-You can manually delete all the files that belong to a modular pipeline. However, Kedro also provides a CLI command to clean up automatically. It deletes the following files when you call `kedro pipeline delete <pipeline_name>`:
-
-
-* All the modular pipeline code in `src/<python_package>/pipelines/<pipeline_name>/`
-* Configuration files `conf/<env>/parameters/<pipeline_name>.yml` and `conf/<env>/catalog/<pipeline_name>.yml`, where `<env>` defaults to `base`. If the files are located in a different config environment, run `kedro pipeline delete <pipeline_name> --env <env_name>`.
-* Pipeline unit tests in `tests/pipelines/<pipeline_name>/`
-
+</details>
 
-```eval_rst
-.. note::  ``kedro pipeline delete`` won't remove the entry from ``pipeline_registry.py`` if you have imported the modular pipeline there. You must remove it manually to clean up, otherwise it will break your project because the import will raise an error.
-```
+![namespaced_params](../meta/images/cook_params.png)
```

Comment:
Can we include a visualisation that also shows that the different parameters are used in the two pipelines? This one only shows two modular pipelines that take the same inputs `input1` and `input2`.